portfolio.css

```diff
@@ -0,0 +1,257 @@
+* {
+	box-sizing: border-box;
+	font-family: 'Nanum Gothic',sans-serif;
+}
+html {
+	font-size: 62.75%;
+}
+a {
+	text-decoration: none;
+	color: inherit;
+}
+.sticky {
```

Comment:
class names should describe what something is, not what it looks like. this seems to do neither? its not sticky, or a stick?